### PR TITLE
Fix Jenkins URL rendering in chart notes

### PR DESCRIPTION
**Pull request by felipecrs**
### Which issue this PR fixes

None, I noticed the issue and fixed it directly.

### Checklist

- [x] [DCO](https://github.com/jenkinsci/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] CHANGELOG.md was updated

### CODE REVIEW

**Comment by felipecrs** [comment]
Ok, I also fixed another nitpick, which was the config-as-code URL always using ingress even when it's not set.

**Comment by felipecrs** [comment]
Ok, I also fixed an issue where TLS were not taken into account in the NOTES, when using ingress.

**Comment by felipecrs** [comment]
I don't think the CI failure has anything to do with my PR.

File: charts/jenkins/templates/NOTES.txt [hunk 1]
```diff
@@ -1,34 +1,43 @@
...
+{{- $url = print "https://" .Values.controller.ingress.hostName $prefix -}}
+{{- else -}}
+{{- $url = print "http://" .Values.controller.ingress.hostName $prefix -}}
+{{- end }}
+2. Visit {{ $url }}
 {{- else }}
 2. Get the Jenkins URL to visit by running these commands in the same shell:
-{{- if contains "NodePort" .Values.controller.serviceType }}
+{{- if contains "NodePort" .Values.controller.serviceType -}}
+{{- $url = print "http://$NODE_IP:$NODE_PORT" $prefix }}
```
Comment (torstenwalter) [inline charts/jenkins/templates/NOTES.txt:15]: I think this won't work. NODE_IP and NODE_PORT are set after this line. Also not sure if you would have access to it.

Comment (felipecrs) [reply charts/jenkins/templates/NOTES.txt:15]: This is just a string that will be printed. These two variables aren't supposed to be expanded, they are just snippets for shell commands. This is how it was before as well.

Comment (felipecrs) [reply charts/jenkins/templates/NOTES.txt:15]: Yeah, that's true. If you'd like me to change something let me know.

Comment (felipecrs) [reply charts/jenkins/templates/NOTES.txt:15]: Ok, and I just realized that TLS were not being taken into account for some other cases as well. Pushing a fix.

**Comment by papi83dm** [comment]
@felipecrs How do you rerun the CI, someone updated the the CI list.

**Comment by felipecrs** [comment]
> @felipecrs How do you rerun the CI, someone updated the the CI list.

I just rebase it with `git pull origin main --rebase`.

**Review by torstenwalter — APPROVED** [review]

**Comment by felipecrs** [comment]
Now all that's missing is to update the repository configuration with the new job names.

**Review by timja — APPROVED** [review]

**Pull request merged**